Vtiger/Model/Validator/Constraints/DateValidator.php

```diff
@@ -2,7 +2,7 @@
 
 namespace MauticPlugin\MauticVtigerCrmBundle\Vtiger\Model\Validator\Constraints;
 
-use Symfony\Component\Validator\Context\ExecutionContextInterface;
+use DateTime;
```

Comment:
I don't think you have to import global classes.
In this case you are trying to import class from the same namespace: `MauticPlugin\MauticVtigerCrmBundle\Vtiger\Model\Validator\Constraints`.
More info here: https://www.php.net/manual/en/language.namespaces.fallback.php
> For functions and constants, PHP will fall back to global functions or constants if a namespaced function or constant does not exist.
There is no DateTime class in the current namespace, so PHP will fallback to the global class. But if somebody create a class in the same directory, it will start to use that new class instead of \DateTime and there will be a bug.
I think this is not your intention. You want to use global \DateTime class instead of DateTime class in the current namespace. Therefore I suppose you have to specify it as \DateTime.
```suggestion
```

Reply:
Thanks, I missed it